go.mod

```diff
@@ -65,7 +65,10 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
-require github.com/google/gnostic v0.5.7-v3refs
+require (
```

Comment:
Can you merge this block with the one above?

Reply:
Updated accordingly.